src/main/java/com/hotels/mutantswarm/report/ReportGenerator.java

```diff
@@ -96,7 +96,7 @@ private File resolveReportFolder() {
   }
 
   private File resolveReportFile(File reportFolder) {
-    String reportFileName = String.format(reportFileNameformat, results.getSuiteName());
+    String reportFileName = String.format(reportFileNameformat, "mutation_report");
```

Comment:
Thinking this could be a pain if theres multiple test classes that someone wants to have a report on. Is there no nice way to get the test class name?